src/Equinox/Tracing.fs

```diff
@@ -0,0 +1,107 @@
+module Equinox.Core.Tracing
+
+open System.Diagnostics
+
+let source = new ActivitySource("Equinox")
+
+module Tags =
+    // General tags
+
+    /// The full stream name
+    [<Literal>]
+    let stream_name = "eqx.stream_name"
+    /// The ID of the stream
+    [<Literal>]
+    let stream_id = "eqx.stream_id"
+    /// The category of the stream
+    [<Literal>]
+    let category = "eqx.category"
+    /// The store being used
+    [<Literal>]
+    let store = "eqx.store"
+
+    // Information about loading events
+
+    /// The version of the stream at read time
+    [<Literal>]
+    let read_version = "eqx.read_version"
+    /// The configured batch size
+    [<Literal>]
+    let batch_size = "eqx.batch_size"
+    /// The total number of batches loaded from the store
```

Comment:
if there are two sets of reads due to a need to resync, does it accumulate or overwrite?